tutorials/styleguide.md

```diff
@@ -270,6 +271,48 @@ Provide inline links to relevant information, where appropriate. For example, li
 
 Provide direct links to pages in the Google Cloud Console when you give Cloud Console-based instructions. These _deep links_ save the reader time spent looking for the right page and can save you time writing descriptions of how to find the page. Deep links open the page with the project set to the user's last-used project.
 
+## Markdown usage
+
+This site uses GitHub-flavored Markdown (GFM) dialect when publishing tutorial content. GFM has a few extensions to basic Markdown, but does not support many extensions present in other Markdown dialects. GFM has a detailed [specification](https://github.github.com/gfm), but it is essentially basic Markdown with defined results for certain corner cases, plus the following extensions:
```

Comment:
We don't actually fully support GitHub-flavored Markdown. I would just say that we use plain markdown, with support for top-level code fences only (code fences used in lists don't work).